department-of-veterans-affairs/va.gov-cms
# VACMS-7321: Add "Enforce unique combo office" field to fix duplicated Non-clinical Service

- VACMS-7321:
    - Added new "Enforce unique combo office" (field_enforce_unique_combo_offic) field to the "VAMC Facility Non-clinical Service" content type and configured it.
    - Adjusted the "VAMC Facility Non-clinical Service" content type form display and rearranged the new field.
    - Updated "content_model_bundles.feature" and "content_model_vamc_content_type_fields.feature" Behat features.

## Description

closes #7321 

## Testing done

Passes all tests

## Screenshots

1. Existing **VAMC Facility Non-clinical Service** node: **Billing and insurance - Sheridan VA Medical Center**
![existing-VAMC-facility-non-clinical-node](https://user-images.githubusercontent.com/846871/147978431-4d06ceea-dd43-4423-8a27-54947d12fda3.png)

1. Duplicated **VAMC Facility Non-clinical Service** node blocked on save: 
![new-VAMC-facility-non-clinical-node-save-notice](https://user-images.githubusercontent.com/846871/147979136-339b1e22-2dd2-4578-ab1f-5c0689651ca1.png)


## QA steps

As user administrator
1. Go to a **VAMC Facility Non-clinical Service** node e.g.: `/node/42145/edit` and use this node's data 
2. Create another node **VAMC Facility Non-clinical Service** `/node/add/vha_facility_nonclinical_service` with the same data and click save
3. After trying to save the "duplicated" node you should see the **_There is existing content: Billing and insurance - Sheridan VA Medical Center, with identical values for the following field(s): Facility, Service._** message (see screen shot # 2 above)


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
